test/e2e/network/netpol/model.go

```diff
@@ -40,11 +40,6 @@ type Model struct {
 	DNSDomain      string
 }
 
-// NewWindowsModel returns a model specific to windows testing.
-func NewWindowsModel(namespaces []string, podNames []string, ports []int32, dnsDomain string) *Model {
```

Comment:
Hey Jay,
This fix does not introduce or skip any test cases, for some cases that test policy on TCP, it will check whether the rules take effect for the other protocol, some test cases create the rules onTCP traffic only, to verify the rules, they will check whether the other protocols are denied/allowed, e.g. UDP, but the pods contains only TCP containers/services, so TIMEOUT occurs for UDP test, this change just gives a chance to let the UDP check work.

Reply:
it introduces UDP to the windows tests b/c now we use the default